universal-login-react/src/ui/commons/WaitingFor.tsx

```diff
@@ -38,7 +59,20 @@ export const WaitingForTransaction = ({action, relayerConfig, transactionHash, c
   return (
     <div className="universal-login-waiting-for-transaction">
       <div className={getStyleForTopLevelComponent(className)}>
-        {relayerConfig ? renderWaitingForTransaction(action, relayerConfig, transactionHash, children) : <Spinner className="waiting-for-spinner" />}
+        {relayerConfig ? renderWaitingForTransaction({action, relayerConfig, transactionHash, children}) : <Spinner className="waiting-for-spinner" />}
+      </div>
+    </div>
+  );
+};
+
+export const WaitingForOnRampProvider = ({className, onRampProviderName}: WaitingForOnRampProviderProps) => {
```

Comment:
Move to `TopUp/Fiat/WaitingForOnRampProvider`

Reply:
✅